clubs/management/commands/extract_testimonials.py

```diff
@@ -0,0 +1,36 @@
+import re
+
+import bleach
+from django.core.management.base import BaseCommand
+
+from clubs.models import Club, Testimonial
+from clubs.utils import clean
+
+
+class Command(BaseCommand):
+    help = 'Extract club testimonials from descriptions and remove testimonials from descriptions.'
+
+    def handle(self, *args, **kwargs):
+        for club in Club.objects.all():
+            if club.description:
+                match = re.match(r'(.*)From members of ([^:]+)[:;](.*)', club.description, re.M | re.I | re.S)
```

Comment:
Is this like a common term we have in club descriptions? If so might be worth putting in a comment explaining that that's the case or giving an example—I always find regexps hard to parse from a readability POV

Reply:
This is the format that @valenciafu imported all of the testimonials in, and it's actually super consistent except for one out of ~90 cases where someone accidentally used a semicolon instead of a colon.

I expect to run this script exactly once and then never again, but I'll document the regex for future purposes!